CHANGELOG/CHANGELOG-1.14.md

```diff
@@ -0,0 +1,86 @@
+# Changelog 1.14
+
+## 1.14.0 - 2023-10-27
+
+### Added
+
+- Add rate-limit config to Egress to specify the rate limit of north-south egress traffic of this Egress. ([#5425](https://github.com/antrea-io/antrea/pull/5425), [@GraysonWu])
+- Add `IPAllocated` and `IPAssigned` conditions to Egress status to improve Egress visibility. ([#5282](https://github.com/antrea-io/antrea/pull/5282), [@AJPL88] [@tnqn])
+- Add goroutine stack dump in `SupportBundle` for both Antrea Agent and Antrea Controller. ([#5538](https://github.com/antrea-io/antrea/pull/5538), [@aniketraj1947])
+- Add "X-Load-Balancing-Endpoint-Weight" header to AntreaProxy Service healthcheck. ([#5299](https://github.com/antrea-io/antrea/pull/5299), [@hongliangl])
+- Add log rotation configuration in Antrea Agent config for audit logs. ([#5337](https://github.com/antrea-io/antrea/pull/5337) [#5366](https://github.com/antrea-io/antrea/pull/5366), [@antoninbas] [@mengdie-song])
+- Add GroupMembers API Pagination support to Antrea Go clientset. ([#5533](https://github.com/antrea-io/antrea/pull/5533), [@qiyueyao])
+- Add Namespaced Group Membership API for Antrea Controller. ([#5380](https://github.com/antrea-io/antrea/pull/5380), [@qiyueyao])
+- Support Pod secondary interfaces on VLAN network, and add OVS bridge creation for the secondary network. ([#5341](https://github.com/antrea-io/antrea/pull/5341) [#5365](https://github.com/antrea-io/antrea/pull/5365) [#5279](https://github.com/antrea-io/antrea/pull/5279), [@jianjuns])
```

Comment:
Probably remove ", and add OVS bridge creation for the secondary network"

Reply:
Removed